### ISSUE TITLE
Add validation to address input

### ISSUE THREAD

**Issue by rscottjohnson** [merinogeospatial/know-your-vote#38]
### Acceptance Criteria
- [ ] Add validation to Street, City, and State input form fields
- [ ] Provide defaults in the cases where incomplete data is provided (example, empty fields)
- [ ] Message for states where there is only one rep, and no data is returned (maybe this is an `else` case where general data of some kind is returned?)

**Comment by merinogeospatial** [comment]
Good call, Scott. I was thinking about validation on the search field(s) today but didn't even think of defaults... maybe we can have it required to have everything filled out before the submit button becomes active? I do like the idea about defaults as well.
